.travis.yml

```diff
@@ -1,10 +1,11 @@
 language: python
-python: 3.5
-install: 
+python: 3.6
+cache: pip
+install:
   - ./setup
   - pip install coveralls
-script: 
+script:
   - coverage run rosie.py test
-
+  - python rosie.py run federal_senate
```

Comment:
so this is basically a workaround to not having journey tests? and if so wouldn't it be wiser to not do this and open up an issue "Create journey tests for federal senate"?